bin/verifyExports.js

```diff
@@ -0,0 +1,37 @@
+const index = require('../src/components/index')
+const findAllFilesInDirectories = require('./helpers/findAllFilesInDirectories')
+
+function verifyExports(files = nonspecJsFiles()) {
+  return files
+}
+
+export default verifyExports
+
+function nonspecJsFiles() {
+  const jsFilesWithSpec = findAllFilesInDirectories(['src/components'], '.js')
+  const jsFiles = [...jsFilesWithSpec].filter((file) => {
+    return !(
+      file.includes('spec') ||
+      file.includes('test') ||
+      file.includes('index')
+    )
+  })
+
+  const componentsFromFiles = jsFiles.map((file) =>
+    file
+      .split('/')
+      .slice(-1)[0]
```

Comment:
what's `.slice(-1)[0]` doing here exactly?

Reply:
Also, if you're just trying to get `FooComponent` from `path/to/FooComponent.js` you could do
`path.basename('path/to/FooComponent.js', '.js')`

See https://nodejs.org/api/path.html#path_path_basename_path_ext

I'd prefer that over tricky splitting and array indicing

Reply:
ah Yeah this is the business. thank you